<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DeepakHP95" failures="2" tests="9" name="Default test" time="158.359" errors="0" timestamp="2025-10-24T23:21:10 IST">
  <testcase classname="com.runner.NaukriRunner" name="validLogin" time="3.794"/>
  <testcase classname="com.runner.NaukriRunner" name="profileUpdatePage" time="1.807"/>
  <testcase classname="com.runner.NaukriRunner" name="updateResume" time="3.392"/>
  <testcase classname="com.runner.NaukriRunner" name="resumeHeadlineEdit" time="0.626"/>
  <testcase classname="com.runner.NaukriRunner" name="keySkills" time="0.678"/>
  <testcase classname="com.runner.NaukriRunner" name="employeementSection" time="1.577"/>
  <testcase classname="com.runner.NaukriRunner" name="educationDetailsSection" time="49.034">
    <failure type="java.lang.AssertionError" message="ERROR: OCCURE DURING CLICKING THE ELEMENT USING JAVASCRIPT EXECUTOR">
      <![CDATA[java.lang.AssertionError: ERROR: OCCURE DURING CLICKING THE ELEMENT USING JAVASCRIPT EXECUTOR
at org.junit.Assert.fail(Assert.java:89)
at com.base.Base_Class.elementClickByJavaScript(Base_Class.java:443)
at com.pageobjectmodel.EducationDetailsSection.educationDetail(EducationDetailsSection.java:92)
at com.runner.NaukriRunner.educationDetailsSection(NaukriRunner.java:66)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- educationDetailsSection -->
  <testcase classname="com.runner.NaukriRunner" name="itSkillsSection" time="1.295"/>
  <testcase classname="com.runner.NaukriRunner" name="profileSummarySection" time="90.236">
    <failure type="java.lang.AssertionError" message="ERROR: OCCURE DURING VISIBILITY OF ELEMENT LOCATED WAIT">
      <![CDATA[java.lang.AssertionError: ERROR: OCCURE DURING VISIBILITY OF ELEMENT LOCATED WAIT
at org.junit.Assert.fail(Assert.java:89)
at com.base.Base_Class.waitForVisibilityOfElement(Base_Class.java:311)
at com.pageobjectmodel.ProfileSummarySection.profileSummary(ProfileSummarySection.java:48)
at com.runner.NaukriRunner.profileSummarySection(NaukriRunner.java:75)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- profileSummarySection -->
</testsuite> <!-- Default test -->
